src/css-image-value.js

```diff
@@ -0,0 +1,39 @@
+(function(internal, scope) {
+
+  function CSSImageValue(image) {
+    if (!(image instanceof Image)) {
```

Comment:
CSSImageValue should take a string URL: https://drafts.css-houdini.org/css-typed-om/#imagevalue-objects